org.kde.kid3.json

```diff
@@ -0,0 +1,100 @@
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "kid3",
+    "rename-icon": "kid3",
+
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+
+        /* Required to load music metadata */
+        "--share=network",
+
+        /* Required for input-output */
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-music",
+        "--env=TMPDIR=/var/tmp",
+
+        /* For MPRIS */
+        "--own-name=org.mpris.MediaPlayer2.kid3"
+    ],
+
+    "cleanup": [
+        "/man",
+        "/share/man",
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        /* For support in both GNOME and KDE, Kid3 has independent components for KDE and Qt
+           Since we build against the KDE runtime, we don't need the -Qt metadata */
+        "/share/**/*-qt.*",
+        "/share/icons/**/**/**/*-qt.*"
+    ],
+
+    "modules": [
+        {
+            "name": "chromaprint",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/acoustid/chromaprint/archive/v1.5.0.tar.gz",
+                    "sha256": "5c8e0d579cb3478900699110aa961c1552a422a18741cf67dd62136b1b877c7b"
+                }
+            ]
+        },
+        {
+            "name": "taglib",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DWITH_MP4=ON",
+                "-DWITH_ASF=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://taglib.org/releases/taglib-1.11.1.tar.gz",
+                    "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+                }
+            ]
+        },
+        {
+
+            "name": "kid3",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DWITH_APPS=KDE;Qt;CLI",
+                "-DWITH_ID3LIB=OFF",
+                "-DDOCBOOK_XSL_DIR=docbook-xsl"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kid3/3.8.3/kid3-3.8.3.tar.xz",
+                    "sha256": "3556e7a76d65baf8b4258b955f6c93d7467a02a0b588ff7249c8ab96bc128377"
+                },
+                {
+                    "type": "archive",
+                    "dest": "docbook-xsl",
+                    "url": "https://github.com/docbook/xslt10-stylesheets/releases/download/release/1.79.2/docbook-xsl-1.79.2.tar.gz",
```

Comment:
It's in the SDK:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/19.08/elements/components/docbook-xsl.bst

Example from another app on Flathub:
https://github.com/flathub/org.bunkus.mkvtoolnix-gui/blob/7fdb6843aa734dc91bf523b5453347125741d688/org.bunkus.mkvtoolnix-gui.yaml#L55

Reply:
Sorry, I did not know that. Then it would be better to use the path from the SDK and do without the download.

    -DDOCBOOK_XSL_DIR=/usr/share/xml/docbook/xml/xsl-stylesheets

Another thing: I noticed that "Move to Trash" from the file list context menu does not work from the flatpak sandbox, so I changed its implementation to use the KIO trash() method, which works. You could use commit https://invent.kde.org/kde/kid3/-/commit/f7c87356def85ea96efbb923f402a9e9a8d01973.patch as a patch.